client/my-sites/exporter/guided-transfer-options.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Button from 'components/forms/form-button';
+
+export default React.createClass( {
+	displayName: 'GuidedTransferOptions',
+
+	render: function() {
+		return (
+			<div>
+				<CompactCard>
+					<div className="exporter__guided-transfer-options-header-left">
```

Comment:
It would be better to use class names that describe the content of the elements, rather than the presentation.